src/_gettsim/parameters/ges_rente.yaml

```diff
@@ -973,23 +976,16 @@ altersgrenze_besonders_langj_versicherte:
       upper_threshold: inf
       rate_linear: 0
       intercept_at_lower_threshold: 65
-  2014-07-01:
+  2014-07-01:  # See comment above *
```

Comment:
Leave as is, no inline comments, please. 

Are we sure that the results are correct on, say, 1 August 2014? Just wondering about the granularity here since thresholds are based on years but introduction was at the monthly level.

Reply:
To calculate the eligibility for a given date it should work -  and I think this is the main function at the moment.
(For the details we should talk really , there is just too many layers of thinking right know to provide relevant answers in written comment.)

Reply:
Great. Let's see about that once we have merged #495 and potentially other larger changes.